docs/user_guide.md

```diff
@@ -248,3 +249,71 @@ If you are using a self-signed cert, make sure to copy the CA cert into ```/etc/
+![browse project](img/clair_not_ready2.png)
+
+Once the database is ready, an overall database updated timestamp will be shown in the **'Vulnerability'** tab of **'Configuration'** section under **'Administration'**. Click on the timestamp drop-down list, the timestamps of different namespaces are listed.
+![browse project](img/clair_ready.png)
+
+**Scan Image** 
+
+Enter your project and locate the specified repository. Expand the tag list via clicking the arrow icon on the left side. For each tag there will be an 'Vulnerability' column to display vulnerability scanning status and related information. You can click on the vertical ellipsis to open a popup menu and then click on 'Scan' to start the vulnerability analysis process. 
+![browse project](img/scan_menu_item.png)
+**NOTES: Only the users with 'Project Admin' role have the privilege to launch the analysis process.**
+
+The analysis process may have the following status that are indicated in the 'Vulnerability' column:
+* **Not Scanned:** The tag has never been scanned.
+* **Queued:** The scanning task is scheduled but not executed yet.
+* **Scanning:** The scanning process is in progress.
+* **Error:** The scanning process failed to complete.
+* **Complete:** The scanning process was successfully completed.
+
+For the **'Not Scanned'** and **'Queued'** statuses, a text label with status information is shown. For the **'Scanning'**, a progress bar will be displayed.
+If an error occurred, you can click on the **'View Log'** link to view the related logs.
+![browse project](img/log_viewer.png)
+
+If the process was successfully completed, a result bar is created. The width of the different colored sections indicates the percentage of features with vulnerabilities for a particular severity level.
+* **Color Red:** **High** level of vulnerabilities
+* **Color Orange:** **Medium** level of vulnerabilities
+* **Color Yellow:** **Low** level of vulnerabilities
+* **Color Grey:** **Unknown** level of vulnerabilities
+* **Color Green:** **None** vulnerabilities
+![browse project](img/bar_chart.png)
+
+Move the cursor over the bar, a tooltip with summary report will be displayed. Besides showing the total number of features with vulnerabilities and the total number of features in the scanned image tag, the report also lists the counts of features with vulnerabilities of different severity levels. The completion time of the last analysis process is shown at the bottom of the tooltip.
+![browse project](img/summary_tooltip.png)
+
+Click on the tag name link, the detail page will be opened. Besides the information about the tag, all the vulnerabilities found in the last analysis process will be listed with the related information. You can order or filter the list by columns.
+![browse project](img/tag_detail.png)
+
+**NOTES: You can initiate the vulnerability analysis for tag at anytime you want as long as the status is not 'Queued' or 'Scanning'** 

```

Comment:
vulnerability analysis for tag
-->
vulnerability analysis for a tag

add "." at the end of the line.